app/components/dashboard/moab_storage_roots_component.html.erb

```diff
@@ -28,23 +27,27 @@
             <td><%= info[2] %></td><%# average size %>
             <td class="text-end"><%= info[3] %></td><%# moab count %>
             <td class="text-end"><%= info[4] %></td><%# ok status %>
-            <% info[5..].map do |val| %>
+            <% info[5..-2].map do |val| %>
```

Comment:
this whole business with the storage_root_info structure is nasty and I made a ticket to fix it